### DISCUSSION TITLE
Pull based model

### DISCUSSION THREAD

**Discussion by patil-kshitij** [open-telemetry/opentelemetry-collector#5553] [Q&A]
Can we initiate pull based approach for traces, metrics and logs in open-telemetry collector?
Also, can we initiate collecting of host-metrics from remote hosts in open-telemetry collector in  pull based model ?

**Comment by jpkrohling** [comment]
I remember @bwplotka talking about this in a couple of situations. I also directed someone else to him some time ago, to share ideas and thoughts, but I don't know what's the outcome. 

@bwplotka, has there been anything concrete out of those discussions?
